jib-gradle-plugin/README.md

```diff
@@ -208,6 +208,7 @@ Property | Type | Default | Description
 --- | --- | --- | ---
 `appRoot` | `String` | `/app` | The root directory on the container where the app's contents are placed. Particularly useful for WAR-packaging projects to work with different Servlet engine base images by designating where to put exploded WAR contents; see [WAR usage](#war-projects) as an example.
 `args` | `List<String>` | *None* | Additional program arguments appended to the command to start the container (similar to Docker's [CMD](https://docs.docker.com/engine/reference/builder/#cmd) instruction in relation with [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#entrypoint)). In the default case where you do not set a custom `entrypoint`, this parameter is effectively the arguments to the main method of your Java application.
+`classpath` | `List<String>` | `/app/resources`, `/app/classes`, `/app/libs/* | The Java classpath to use.
```

Comment:
Should this be `extraClasspath`? Also the explanation seems too vague. Closing tick missing for `/app/libs/*` too.

 BTW, we update the doc after making a release.

Reply:
Good catch.  I'll submit separately.